docs/upgrade.html

```diff
@@ -70,6 +70,11 @@
     <li>Added user headers support through a new Headers interface providing user headers read and write access.</li>
     <li>ProducerRecord and ConsumerRecord expose the new Headers API via <code>Headers headers()</code> method call.</li>
     <li>ExtendedSerializer and ExtendedDeserializer interfaces are introduced to support serialization and deserialization for headers. Headers will be ignored if the configured serializer and deserializer are not the above classes.</li>
+    <li>A new config,<code>group.initial.rebalance.delay.ms</code>, was introduced.
+        This config specifies the time, in milliseconds, that the <code>GroupCoordinator</code> will delay the initial consumer rebalance.
+        The rebalance will be further delayed by the value of <code>group.initial.rebalance.delay.ms</code> as new members join the group, up to a maximum of <code>max.poll.interval.ms</code>.
+        The default value for this is 3 seconds.
```

Comment:
Thanks. Is it worth a note that it's a good idea to set this to 0 in tests and dev?